### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by astroboyReloaded**
- `create-react-app`.
- Install **linters**.
- install `react-router-dom`.
- Create **routes**.
- Create `<Books>` and `<Categories>`.
- **Render** list of books.
- Add **Remove** `<button>`.
- Create **AddNew Book** `<form>`.
- Create **Navigation** and `<Layout>`.

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <section>
+          <ul>
+            {bookList.map((book) => (
+              <li key={book.title}>
+                <article>
+                  <span>{book.category}</span>
+                  <h1>{book.title}</h1>
+                  <data>{book.author}</data>
+                  <button type="button">Remove</button>
+                </article>
```
Comment (emmiiorji) [inline src/components/Books.js]: As specified in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/be3b13a85ca425c0bf0eeb1f130b6a52e36deb4f/bookstore/project_init_project_wth_comps_v3_0.md?plain=1#L42), kindly create a new component for book and use it within the books component.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @astroboyReloaded, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Project was initialized using [Create React App](https://create-react-app.dev/)
- [React Router](https://github.com/remix-run/react-router/tree/dev/examples/basic) has been correctly set up

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+              </li>
+            ))}
+          </ul>
+        </section>
+        <section>
+          <h2>ADD NEW BOOK</h2>
+          <form>
+            <input type="text" placeholder="Book title" />
+            <input type="text" placeholder="autor" />
+            <button type="button">ADD BOOK</button>
```
Comment (emmiiorji) [inline src/components/Books.js]: Kindly make this button of type "submit" in order to be able to submit the form.

**Review by franclobo — APPROVED** [review]
Hi @astroboyReloaded,

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Use React Router ✔️
- Use React Components ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by astroboyReloaded** [comment]
Thank you!

**Pull request merged**